src/index.js

```diff
@@ -34,20 +33,21 @@ Sentry.init({
   dsn: process.env.SENTRY_DSN,
   environment: process.env.THAT_ENVIRONMENT,
   release: process.env.SENTRY_VERSION || defaultVersion,
+  debug: process.env.NODE_ENV === 'development',
+});
+
+Sentry.configureScope(scope => {
```

Comment:
Is this not in the other api's?

Reply:
which part? I'm not sure what you're referring too.

Reply:
Sentry.configureScope(). I thought it was missing from the first PR I reviewed. I'll have to go back...